Added JSON string support to import multiple children objects, instead of using a colon-separated import statement (DCNE-510) (#1375)
CiscoDevNet/terraform-provider-aci: docs/resources/rest_managed.md

Author: Done!

An existing object can be [imported](https://www.terraform.io/docs/import/index.html) into this resource via its distinguished name (DN), via the following command:

```
terraform import aci_rest_managed.example_tenant `{ "parentDn": "uni/tn-{name}", "childRns": ["rsTenantMonPol", "annotationKey-[{key}]", "{child-n}"] }`
```

> !> The use of the colon-separated format to import children for the resource will be deprecated in the next release. Please use a JSON format string to import children, instead of using a colon-separated import statement.

Reviewer: ```suggestion
!> The use of the colon-separated format to import children for the resource is deprecated and will be removed in the next release. Please use the JSON format string shown above to import children, instead of using a colon-separated import statement.
```

Author: Done!

> !> The use of the colon-separated format to import multiple child resources will be deprecated in the next release. Please use a JSON format string to import multiple children, instead of using a colon-separated import statement.

Reviewer 2: ```suggestion
!> The use of the colon-separated format to import children for the resource will be deprecated in the next release. Please use a JSON format string to import children, instead of using a colon-separated import statement.
```

```
terraform import aci_rest_managed.example_tenant `{ "parentDn": "uni/tn-{name}", "childRns": ["rsTenantMonPol", "annotationKey-[{key}]", "{child-n}"] }`
```

Starting in Terraform version 1.5, an existing object can be imported using [import blocks](https://developer.hashicorp.com/terraform/language/import) via the following configuration:

> ```child

Reviewer 2: ```suggestion
```
```

```
terraform import aci_rest_managed.example_tenant uni/tn-{name}

or

terraform import aci_rest_managed.example_tenant `{ "parentDn": "uni/tn-{name}" }`

> ```

Reviewer: ```suggestion

\```
terraform import aci_rest_managed.example_tenant uni/tn-{name}
\```
To support more complex scenario, it can also be imported using a JSON representation using the following command:
\```
terraform import aci_rest_managed.example_tenant `{ "parentDn": "uni/tn-{name}" }`
\```
```

> When children need to be imported they must be specified by appending them to the distinguished name (DN) with the below format:

Reviewer: ```suggestion
When children need to be imported they must be specified in the JSON object under the "childRns" key as a list of relative name (RN) next to the DN key as shown below:
```

terraform import aci_rest_managed.example_tenant uni/tn-{name}
```

!> The use of the colon-separated format to import multiple child resources will be deprecated in the next release.

> 

Reviewer 2: Should we remove the deprecated example from the documentation, to force people to use the new example?

terraform import aci_rest_managed.example_tenant uni/tn-{name}
```

> To support more complex scenarios, it can also be imported using a JSON representation using the following command:

Reviewer: ```suggestion
To support more complex scenarios, it can also be imported using a JSON representation as shown in the following command:
```

terraform import aci_rest_managed.example_tenant uni/tn-{name}:{child-rn-1},{child-rn-N}
```

> !> The use of the colon-separated format to import children for the resource is deprecated and will be removed in the next release. Please use the JSON format string shown below to import children, instead of using a colon-separated import statement.

Reviewer: ```suggestion
!> The use of the previously supported colon-separated format to import children for the resource is deprecated and will be removed in the next release as it creates issue with DN containing colon. Please use the JSON format string shown above to import children.
```

Move this block after the next import command.

Author: Done!
